tools/c7n_gcp/c7n_gcp/resources/compute.py

```diff
@@ -599,3 +599,51 @@ def get_resource_params(self, model, resource):
                   }}
 
         return result
+
+
+@resources.register('gce-project')
```

Comment:
was thinking it would be good to rename to compute-project

Reply:
fixed